client/disputes/index.js

```diff
@@ -36,42 +42,53 @@ const headers = [
 		key: 'status',
 		label: __( 'Status', 'woocommerce-payments' ),
 		required: true,
+		isLeftAligned: true,
 	},
 	{
 		key: 'reason',
 		label: __( 'Reason', 'woocommerce-payments' ),
 		required: true,
+		isLeftAligned: true,
 	},
 	{
 		key: 'source',
 		label: __( 'Source', 'woocommerce-payments' ),
 		required: true,
+		cellClassName: 'payment-source',
```

Comment:
IMO, we can keep it more generic like: `is-center-aligned` so that we don't duplicate the same styling for other columns to be centrally aligned in the future.

Reply:
Brilliant idea! Updated 9f875e6